### PR TITLE
Restaurants pages added

### PR DESCRIPTION
**Pull request by sadikekin**
I have added main and show page about restaurants
screen shot 2017-10-12 at 9 53 03 am
screen shot 2017-10-12 at 9 52 49 am

### CODE REVIEW

**Comment by sadikekin** [comment]
Guys, did some improvement about directory design. Added Restaurant Show and Index page.

**Comment by Alpkant** [comment]
Why did you remove the style.css ? Also , one other thing why the navbar is pink ?

**Review by Alpkant — COMMENTED** [review]
Light pink looks awkward on nav bar it could be red. Style css shouldn't be removed we can use it as a common css file.

**Comment by sadikekin** [comment]
Yeah, because we don't need style.css. It was for professors temple. We are not using it. 
Last night we discussed it in our WhatsApp group. If we use red, costumes feel more hungry

**Comment by sadikekin** [comment]
@Alpkant can you give me hex form of color so I can add

**Comment by sadikekin** [comment]
So I did changes according to @Alpkant s feedback

**Review by Alpkant — APPROVED** [review]
Okay , now it looks nice. Good job.

**Review by onatsahin — APPROVED** [review]

File: server.py [hunk 1]
```diff
@@ -10,7 +10,17 @@
 @app.route('/')
 def home_page():
     now = datetime.datetime.now()
-    return render_template('home.html', current_time=now.ctime())
```
Comment (Bekci) [inline server.py]: Why do you changed route page? I supposed to do the home page so do I need to change home.html to index.html?

**Review by Bekci — COMMENTED** [review]
Why did you changed route page? I supposed to do the home page so do I need to change home.html to index.html?

File: server.py (hunk 1 above)
Comment (sadikekin) [reply server.py]: Hello there Bekci,
Yes

Comment (Alpkant) [reply server.py]: Yes we have changed organization of folders and html pages our homepage is now index.html .You should change it as well.

Comment (Bekci) [reply server.py]: Okay everything is clear now. Thanks!

**Review by Bekci — APPROVED** [review]

**Pull request merged**